recipe/meta.yaml

```diff
@@ -38,6 +38,7 @@ requirements:
     - vc 9  # [win and py27]
     - vc 10  # [win and py34]
     - vc 14  # [win and py35]
+    - pthread-stubs
```

Comment:
Yeah ... I had `xorg-libpthread-stubs` as a runtime dep of `xorg-libxcb`, but when I switched over to use the same names as the `defaults` versions of those packages, I didn't bring the dep over, since `pthread-stubs` (in both incarnations) literally only provides a single pkg-config file. So a couple of the X.org packages will likely need to have it added as a build-time dep since pkg-config will complain otherwise. It didn't seem like something that needed to be proactively dealt with since the fix is so trivial.